data/data-pipeline/data_pipeline/tests/test_score_output.py

```diff
@@ -0,0 +1,252 @@
+from pathlib import Path
+import pandas as pd
+import pytest
+import data_pipeline.score.field_names as field_names
+from data_pipeline.utils import get_module_logger
+from data_pipeline.config import settings
+from data_pipeline.score.score_narwhal import ScoreNarwhal
+
+logger = get_module_logger(__name__)
+
+
+# this is so slow?
+@pytest.fixture
+def final_score_df():
+    return pd.read_csv(
+        settings.APP_ROOT / "data" / "score" / "csv" / "full" / "usa.csv",
+        dtype={field_names.GEOID_TRACT_FIELD: str},
+    )
+
+
+# when should things be fixtures??
```

Comment:
General comment -- when should things be fixtures vs. constants? I've erred on the side of fixtures because that is how it looks in our codebase... but interested in comments and thoughts

Reply:
@esfoobar-usds I personally wouldn't make most of these fixtures --- I use fixtures for things I'm calculating (data or complicated mocks) that I want to reset with each test invocation, and I would put the rest just as constants I import to reduce the mental overhead with figuring out where a given name comes from (since fixtures have a little bit of magic to them in how they use DI). What's your-all's overall style though?

Reply:
This is SUPER helpful, @mattbowen-usds. TBH I have no idea on our structure, which is why (again, I am not an engineer!) I don't think I am the best choice for designing all of these tests. @esfoobar-usds could you please provide guidance?